src/pages/guides/quickstart/index.md

```diff
@@ -15,43 +15,58 @@ keywords:
   - Getting started
   - Browser support
 title: Quickstart Guide
-description: This is the quickstart guide for Adobe Express Embed SDK
+description: This is the Quickstart Guide for Adobe Express Embed SDK
 contributors:
   - https://github.com/amandahuarng
   - https://github.com/nimithajalal
+  - https://github.com/undavide
 ---
 # Quickstart Guide
 
-This guide explains how to start using the Adobe Express Embed SDK in your own application.
+In this guide, you will learn how to get started with the Adobe Express Embed SDK.
 
 ## Step 1: Get an API Key
 
-Follow the below steps to get an API Key.
+API keys—one for each project—are required to use the Adobe Express Embed SDK. You can get one logging to your [Adobe Developer Console](https://developer.adobe.com/console); if you need a refresher on the Console, you can read more about it [here](https://developer.adobe.com/developer-console/docs/guides/getting-started/).
 
-1. Navigate to the [Developer Console](https://developer.adobe.com/console).
-2. Select **APIs and services**.
-3. Search for **Adobe Express Embed SDK**.
-4. Click **Create project**.
+![](../images/quickstart_console.png)
 
-On the **Configure API** section,
+Select **APIs and services** from the top navigation bar; you'll see there a list of the available ones. Find the **Adobe Express Embed SDK** card and click the **Create project** button.
```

Comment:
Can we rewrite this in steps as above? 

1. Navigate to the [Developer Console](https://developer.adobe.com/console).
2. Select **APIs and services**.
3. Search for **Adobe Express Embed SDK**.
4. Click **Create project**.

Reply:
Also, let us mention here as additional information that the project name will be referred to as `appName: <APP_NAME>` in the further code samples.

Reply:
I'll add that later on, right before the Step 2 heading, because it looks like you are not asked to name the project—you can only re-name it after it's created. There's already a small section on changing the domains, I'll add renaming the project and the `appName` ref there.